calcom/cal.com
# fix: Org booking pages link fixes for Routing Forms and Event type

## What does this PR do?

- Fixes booking links for Routing Forms and EventType(from Single view) for an organization.

- Fixes link and avatar(missed in avatars PR) on members page 
![image](https://github.com/calcom/cal.com/assets/1780212/1e0023c5-fd42-4c1c-a124-69f80c876c03)



## Requirement/Documentation
All booking pages are supposed to be on organisation domain(including Routing Forms)

## Type of change




## How should this be tested?
  - No Env Vars needed to test locally.
  - Simply create an event-type and a routing-form after logging in as an organization member.
  - Go to event-type e.g. /event-types/1 and click on "Preview" and check that URL is of org domain(e.g. org.cal.com)
![image](https://github.com/calcom/cal.com/assets/1780212/5df940f4-3523-4cdc-aaea-d018c75ec2df)
- Similarly, to Routing Form e.g./apps/routing-forms/forms/1 and click on "Preview" and check that the URL is of org domain.
 ![image](https://github.com/calcom/cal.com/assets/1780212/abc24d79-0e55-46f1-b820-b0c39372dfbc)
- Do a repeat test with EventType and Routing Form for a non-org user and see that now the URL is app URL and not org url.



## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
